shared/shot.js

```diff
@@ -603,18 +603,19 @@ class AbstractShot {
 AbstractShot.prototype.REGULAR_ATTRS = (`
 origin fullUrl docTitle userTitle createdDate favicon images
 siteName openGraph twitterCard documentSize
-fullScreenThumbnail abTests
+thumbnail abTests
 `).split(/\s+/g);
 
 // Attributes that will be accepted in the constructor, but ignored/dropped
 AbstractShot.prototype.DEPRECATED_ATTRS = (`
 microdata history ogTitle createdDevice head body htmlAttrs bodyAttrs headAttrs
 readable hashtags comments showPage isPublic resources deviceId url
+fullScreenThumbnail
```

Comment:
I didn't realize we had never deprecated fullScreenThumbnail (which was from when we were doing more expansive data capturing). Anyway, good to have it gone.